spec/views/sufia/admin/admin_sets/_form.html.erb_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+RSpec.describe 'sufia/admin/admin_sets/_form.html.erb', type: :view do
+  let(:admin_set) { create(:admin_set) }
+  let!(:workflow) { Sipity::Workflow.create!(name: "default", label: "default") }
+  let!(:permission_template) { Sufia::PermissionTemplate.find_or_create_by(admin_set_id: admin_set.id, workflow_name: 'default') }
+  before do
+    @form = Sufia::Forms::AdminSetForm.new(admin_set, permission_template)
```

Comment:
This could be `instance_double(Sufia::Forms::AdminSetForm)` and then we wouldn't need to persist all the objects in Fedora/RDBMS